.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+<!--
+Thanks for sending a pull request!  Here are some tips for you:
+  1. If this is your first time, please read our contributor guidelines: https://github.com/delta-io/delta/blob/master/CONTRIBUTING.md
+  2. If the PR is unfinished, add '[WIP]' in your PR title, e.g., '[WIP] Your PR title ...'.
+  3. Be sure to keep the PR description updated to reflect all changes.
+  4. Please write your PR title to summarize what this PR proposes.
+  5. If possible, provide a concise example to reproduce the issue for a faster review.
+  6. If applicable, include corresponding issue number in the PR title and link it in the body.
+-->
+## Related Issue
+
+<!-- Link related issue
+For instance,
+  [#ISSUE NUMBER] Your PR title
+-->
```

Comment:
Could we remove this section entirely? I added a suggestion for adding this below and now it's in 6) above. I don't think a separate section for just this is necessary

Reply:
Ok sure.